SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -56,6 +57,9 @@ void FSpatialGDKEditorModule::StartupModule()
 		FModuleManager::LoadModuleChecked<IAutomationControllerModule>(TEXT("AutomationController"));
 	IAutomationControllerManagerPtr AutomationController = AutomationControllerModule.GetAutomationController();
 	AutomationController->OnTestsComplete().AddLambda([]() {
```

Comment:
Ah I didn't realize this was in the editor module. Maybe in the future we should move this to the FunctionalTesting module and have a conditional dependency on the UnrealEditor.